src/view/components/site-selector/dot-site-selector-component.ts

```diff
@@ -38,4 +42,44 @@ export class SiteSelectorComponent extends BaseComponent {
 
         });
     }
+
+    /**
+     * Filter the users displayed in the dropdown by comparing if
+     * the user name characters set on the drowpdown search box matches
+     * some on the user names set on the userlist variable loaded on the
+     * ngOnInit method
+     *
+     * @param event - The event with the query parameter to filter the users
+     */
+    filterSites(event): void {
+        this.filteredSitesResults = [];
+        for(let i = 0; i < this.sites.length; i++) {
+            let site = this.sites[i];
+            if(site.hostName.toLowerCase().indexOf(event.query.toLowerCase()) >= 0) {
+                this.filteredSitesResults.push({
+                    label: site.hostName,
+                    value: site.identifier,
+                });
+            }
+        }
```

Comment:
better use:

this.filteredSitesResults = this.sites
      .filter( site => site.hostName.toLowerCase().indexOf(event.query.toLowerCase()) >= 0)
      .map( site => {
                     label: site.hostName,
                     value: site.identifier,
                 } );